src/overlaybd/tar/erofs/test/test.cpp

```diff
@@ -250,6 +252,111 @@ TEST_F(ErofsTest, tar_meta) {
     delete tar_idx;
 }
 
+class ErofsPax : public ::testing::Test {
+protected:
+    photon::fs::IFileSystem *host_fs;
+    std::string workdir = "/tmp/pax_test";
+    std::string src_url = "https://github.com/salvete/erofs-imgs/raw/main/pax.tar";
```

Comment:
Can we just keep gzipped data in the testcase itself so that it won't rely on the network?
see https://github.com/containerd/overlaybd/pull/353/files#diff-eb6cf39198971c1db93f32d0f8bcb91f32e5cc73dd97fb29585bfe40ea90a2c8L198

Reply:
OK, I will give it a try.

Reply:
Done.